src/core/mac/mac_frame.hpp

```diff
@@ -528,7 +528,7 @@ class Frame : public otRadioFrame
      * @returns TRUE if the Src PanId is present, FALSE otherwise.
      *
      */
-    bool IsSrcPanIdPresent(uint16_t aFcf) const;
+    static bool IsSrcPanIdPresent(uint16_t aFcf);
```

Comment:
Shall we move it into private, the same position as `IsDstPanIdPresent`? This function is actually not used externally.